frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -126,35 +120,23 @@ export const insightLogic = kea<insightLogicType>([
                     }
                     throw new Error(`Insight "${shortId}" not found`)
                 },
-                updateInsight: async ({ insight, callback }, breakpoint) => {
```

Comment:
This action is setup for generic insight updates, but in reality it's only used when adding and removing it from the dashboard.